packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -3,10 +3,13 @@
  */
 
 import url from 'url';
+
 import {WebexPlugin} from '@webex/webex-core';
 import {deprecated, oneFlight} from '@webex/common';
 import {camelCase, get, set} from 'lodash';
 import backoff from 'backoff';
+import HttpsProxyAgent from 'https-proxy-agent';
```

Comment:
This module can only be used when running in node, it will not work in the browser.

Reply:
What happens when run on the browser? The mercury plugin is utilized by the browser based sdk in multiple places.